### DISCUSSION TITLE
MasVis 音轨比对指南

### DISCUSSION THREAD

**Discussion by giscus** [ZexWoo/Giscus#33] [Announcements]
# MasVis 音轨比对指南

同一首歌可能会有不同的版本，我们要如何科学、细致地比较它们？本文将给出答案。

https://zexwoo.blog/posts/tutorials/masvis/

**Comment by dumbbird** [comment]
我对这个软件很感兴趣！但是。。。。貌似官网已经404了。唉。

**Reply by dumbbird** [reply]
补充：官网的链接改变了～可以更新一下：https://www.lts.a.se/ljudteknik/masvis/

**Reply by ZexWoo** [reply]
谢谢，很快就会更新掉。
